code/modules/clothing/masks/gasmask.dm

```diff
@@ -100,13 +100,13 @@
 	actions_types = list(/datum/action/item_action/toggle_mask, /datum/action/item_action/change_appearance_mask, /datum/action/item_action/toggle_voicechanger)
 	species_fit = list(VOX_SHAPED, GREY_SHAPED)
 	var/list/clothing_choices = list()
-
+//someone who isn't me should really refactor this so it isnt instantiating all the masks in the game on startup
```

Comment:
I'm pretty sure it doesn't, because it isn't reading any lists in `change()`
But even if it doe, just making the list `global` and not generating it if it's non-`null` would fix the infinite loop